WORKSPACE

```diff
@@ -321,5 +323,84 @@ rbe_autoconfig(
 
 load("@build_bazel_integration_testing//tools:repositories.bzl", "bazel_binaries")
 
-#depend on the Bazel binaries, also accepts an array of versions
-bazel_binaries()
+# Depend on the Bazel binaries
+bazel_binaries(versions = ["0.28.1"])
+
+#
+# Support creating Docker images for our node apps #
+# Not used but needed for @e2e_angular_bazel_example//:bazel_integration_test_files target
+#
+
+http_archive(
+    name = "io_bazel_rules_docker",
+    sha256 = "aed1c249d4ec8f703edddf35cbe9dfaca0b5f5ea6e4cd9e83e99f3b0d1136c3d",
+    strip_prefix = "rules_docker-0.7.0",
+    urls = ["https://github.com/bazelbuild/rules_docker/archive/v0.7.0.tar.gz"],
+)
+
+load("@io_bazel_rules_docker//nodejs:image.bzl", nodejs_image_repos = "repositories")
+
+nodejs_image_repos()
+
+#
+# Kubernetes setup, for deployment to Google Cloud
+# Not used but needed for @e2e_angular_bazel_example//:bazel_integration_test_files target
+#
+
+git_repository(
+    name = "io_bazel_rules_k8s",
+    commit = "36ae5b534cc51ab0815c9bc723760469a9f7175c",
+    remote = "https://github.com/bazelbuild/rules_k8s.git",
+    shallow_since = "1545317854 -0500",
+)
+
+load("@io_bazel_rules_k8s//k8s:k8s.bzl", "k8s_defaults", "k8s_repositories")
+
+k8s_repositories()
+
+k8s_defaults(
+    # This creates a rule called "k8s_deploy" that we can call later
+    name = "k8s_deploy",
+    # This is the name of the cluster as it appears in:
+    #   kubectl config view --minify -o=jsonpath='{.contexts[0].context.cluster}'
+    cluster = "_".join([
+        "gke",
+        "internal-200822",
+        "us-west1-a",
+        "angular-bazel-example",
+    ]),
+    kind = "deployment",
+)
+
+#
+# Setup bazel_integration_test repositories
+#
+
+[local_repository(
```

Comment:
we could do this comprehension for the `packages/foo` workspaces too

Reply:
👍